scripts/workspace_cleanup/README.md

```diff
@@ -8,6 +8,15 @@ The scripts in this directory can be used to delete Terra workspaces in bulk. Pr
 
 These scripts use the Terra APIs, so they can only delete workspaces that the running user has access to. If your goal is to view as many workspaces as possible in the project, it's best to make sure the person running the script is a project owner on the Terra project. Note that any workspace covered by an Authorization Domain (AD) will not be able to be deleted (or have cost information returned) by these scripts unless the person running the script is in the workspace's AD.
 
+## Docker image & run command
+To run the scripts in this directory, you can use the Docker image `broadinstitute/horsefish:latest`. \
```

Comment:
do we want to use `latest` or a specific tag?

Reply:
I meant to use a specific tag - I was actually making those updates while you were reviewing =)